plugins/main/public/controllers/management/components/management/statistics/statistics-dashboard-analysisd.js

```diff
@@ -23,206 +21,5 @@ export function WzStatisticsAnalysisd({
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>Syscollector</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-Analysisd-Syscollector'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>Rootcheck</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-Analysisd-Rootcheck'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>SCA</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-Analysisd-SCA'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>Host Info</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-Analysisd-HostInfo'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </div>
-  );
+  return <DashboardAnalysisEngineStatistics isClusterMode={isClusterMode} />;
```

Comment:
**question**: are some props missing? https://github.com/wazuh/wazuh-dashboard-plugins/pull/6542/files#diff-e2122534324fe16fe2d86fbabe7728df8798016a48a3a23d0270819fee99c5d3R43-R45

Reply:
`statistics-dashboard-remoted.js` and `statistics-dashboard-analysisd.js` are no longer used. They are removed.